website/routes.py

```diff
@@ -173,6 +173,8 @@ def make_url_map(app):
 
     process_rules(app, [
 
+        Rule('/home/', 'get', website_views.home, OsfWebRenderer('home.mako')),
```

Comment:
This is a placeholder for the new "dashboard" page, for Caner's work, while the current dashboard is going to be replaced with "My Projects". When a user logs in or goes to osf.io/, this page will be seen instead of dashboard, with a link to "My Projects". Pinging @caneruguz for a better explanation.